### PR TITLE
Swallow ns error if the istio ns pre-exists

### PR DESCRIPTION
**Pull request by irisdingbj**
https://github.com/istio/istio/issues/18948

### CODE REVIEW

File: pkg/manifest/installer.go [hunk 1]
```diff
@@ -321,13 +321,13 @@ func applyManifest(componentName name.ComponentName, manifestStr string, version
 
 		stdoutNs, stderrNs, err := kubectl.Apply(opts.DryRun, opts.Verbose, opts.Kubeconfig, opts.Context, namespace, mns, extraArgs...)
 		stdout += "\n" + stdoutNs
-		stderr += "\n" + stderrNs
+		outerr := stderr + "\n" + stderrNs
```
Comment (ostromart) [inline pkg/manifest/installer.go]: I may be misreading this, but doesn't this lead to the exact same output as before?

Comment (irisdingbj) [reply pkg/manifest/installer.go]: Now we will not the error  error like  `kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply (repeated 1 times)` in the output if the there is no `err` in the namespace creation. We will only report this error if we fail to create the namespace and there is `err` in the namespace creation.

Comment (ostromart) [reply pkg/manifest/installer.go]: I guess what confuses me here is you are replacing:
a += "foo"
with:
b := a + "foo"
Isn't that the same value? I seem to be missing something obvious here.

Comment (irisdingbj) [reply pkg/manifest/installer.go]: @ostromart  Thanks for the review. Seems this already been fixed. So let me close this pr.

**Pull request closed without merging**